$ cargo test
   Compiling rettangolo v0.1.0 (file:///progetti/rettangolo)
    Finished `test` profile [unoptimized + debuginfo] target(s) in 0.32s
     Running unittests src/lib.rs (target/debug/deps/rettangolo-f3dbc7f03c0e31f1)

running 2 tests
test tests::grande_contiene_piccolo ... FAILED
test tests::piccolo_non_contiene_grande ... ok

failures:

---- tests::grande_contiene_piccolo stdout ----

thread 'tests::grande_contiene_piccolo' (6902) panicked at src/lib.rs:31:9:
assertion failed: grande.puo_contenere(&piccolo)
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace


failures:
    tests::grande_contiene_piccolo

test result: FAILED. 1 passed; 1 failed; 0 ignored; 0 measured; 0 filtered out; finished in 0.00s

error: test failed, to rerun pass `--lib`
